packages/polaris-schema/test/utils/executable-schema-creator.test.ts

```diff
@@ -28,33 +28,33 @@ describe('makeExecutablePolarisSchema tests', () => {
     };
 
     test('making polaris schema, returns GraphQLSchema', () => {
-        const polarisSchema = makeExecutablePolarisSchema(typeDefs, resolvers);
+        const polarisSchema = makeExecutablePolarisSchema(false, typeDefs, resolvers);
```

Comment:
add tests with enable federation true (see that everything you checked here for regular schema also works for federated schema